_episodes/11-licensing.md

```diff
@@ -37,6 +38,25 @@ considerations include:
 * Whether the content you are licensing is source code.
 * Whether you want to license the code at all.
 
+You may also want to consider the communities you hope will use and 
```

Comment:
I really like the material you've added here and I really like how you've framed licenses within the communities lens. However, it's adding quite a bit of content to the lesson, which is adding to our already-too-long lesson [see our contributing guidelines](https://github.com/swcarpentry/git-novice/blob/gh-pages/CONTRIBUTING.md). However, we're always looking for new exercises and exercise solutions! I think this is exactly the type of content that could be put into an exercise. Would you mind reframing what you've written here as an exercise (you already have the solutions!) at the end of the section instead?

Reply:
Thank you so much! I just went ahead and changed the format of this chunk of text to an exercise. Please let me know if you'd like me to make any additional changes.

Reply:
I also like the changes, and agree it is better as an exercise.  While you put it in the exercise formatting, it's not written as an exercise.  Can you reframe it as an exercise? 

One way of reframing it would be _Use choosealicense.com to determine the best license for the following types of projects..._ 

List the projects as an ordered list.

List the answers in an answer section, as an ordered lists.

Exercises should allow learners to go through a process to discover the answers themselves.